src/main/resources/view/PersonListCard.css

```diff
@@ -0,0 +1,23 @@
+#tags {
+    -fx-hgap: 7;
+    -fx-vgap: 3;
+}
+
+#tags .label {
+    -fx-text-fill: white;
+    -fx-background-color: #3e7b91;
+    -fx-padding: 1 3 1 3;
+    -fx-border-radius: 2;
+    -fx-background-radius: 2;
+    -fx-font-size: 11;
+}
+
+.cell_big_label {
+    -fx-font-family: "Arial";
+    -fx-font-size: 16px;
+}
+
+.cell_small_label {
+    -fx-font-family: "Arial";
+    -fx-font-size: 13px;
+}
```

Comment:
Should we increase the difference in font size to make it more noticeable?

Reply:
I agree, and perhaps a different font family could be used to increase the user's experience?